utils.py

```diff
@@ -56,6 +56,7 @@ def register(company):
         service_result = json.dumps(service_response)
         service_output = json.loads(service_result)
         # account_id = service_output['account_id']
+        print(service_output)
```

Comment:
I think this is just forgotten debug print (?)

Reply:
I ll move it to log going forward